x/pool-incentives/keeper/grpc_query.go

```diff
@@ -87,6 +87,23 @@ func (q Querier) IncentivizedPools(ctx context.Context, _ *types.QueryIncentiviz
 	lockableDurations := q.Keeper.GetLockableDurations(sdkCtx)
 	distrInfo := q.Keeper.GetDistrInfo(sdkCtx)
 
+	// Add epoch duration to lockable durations if not already present.
+	// This is to ensure that concentrated gauges (which run on epoch time) are
+	// always included in the query, even if the epoch duration changes in the future.
+	epochDuration := q.incentivesKeeper.GetEpochInfo(sdkCtx).Duration
+	epochAlreadyLockable := false
+	for _, lockableDuration := range lockableDurations {
+		if lockableDuration == epochDuration {
+			epochAlreadyLockable = true
+			break
+		}
+	}
+
+	// Ensure that we only add epoch duration if it does not already exist as a lockable duration.
+	if !epochAlreadyLockable {
+		lockableDurations = append(lockableDurations, epochDuration)
+	}
```

Comment:
```suggestion
	epochDuration := q.incentivesKeeper.GetEpochInfo(sdkCtx).Duration
	for _, lockableDuration := range lockableDurations {
		if lockableDuration == epochDuration {
			lockableDurations = append(lockableDurations, epochDuration)	
			break
		}
	}
```

Reply:
This change seems to add epoch time to lockable durations only if it already exists in there - we want the exact opposite no?